community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,329 @@
+
+    a. From the Jenkins machine, run:
+
+```
+hal admin deprecate version --version ${VERSION_TO_DEPRECATE}
+```
+
+   b. Make a PR against correct changelog
+   [here](https://github.com/spinnaker/spinnaker.github.io/tree/master/_changelogs),
+   adding `deprecated` to the list of tags.
+   
+   c. Delete the associated Jenkins project (e.g., Flow_BuildAndValidate_${RELEASE-3}).
+   
+   d. Remove the changelog from the [master gist](https://gist.github.com/spinnaker-release/4f8cd09490870ae9ebf78be3be1763ee).
+   (While logged in as spinnaker-release, click "Edit", scroll to the file, and
+   click "Delete".)
+   
+7. At this point, there should exist the following Flow_BuildAndValidate_* jobs:
+
+    a. Flow_BuildAndValidate_${RELEASE-2} (BUILDING NIGHTLY)
+    
+    b. Flow_BuildAndValidate_${RELEASE-1} (BUILDING NIGHTLY)
+    
+    c. Flow_BuildAndValidate_${RELEASE} (BUILDING NIGHTLY)
+    
+    d. Flow_BuildAndValidate (master, BUILDING NIGHTLY)
+    
+8. Ping the #spinnaker-releases channel to let them know that the new version is
+available.
+
+9. Publish a Spin CLI minor version.
+    
+    a. Each Spin CLI release is tied to a version of Gate. To ensure
+    compatibility, regenerate the Gate Client API.
+    
+    b. From the Gate repository, check out the release branch and follow the
+    [instructions](https://github.com/spinnaker/spin/blob/master/CONTRIBUTING.md#updating-the-gate-api)
+    for updating the generated Gate Client API. Cherry-pick the Gate Client API
+    changes onto the Spin CLI release branch. As of writing, the Swagger
+    Codegen CLI uses 2.3.1; you can get that JAR [here](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar).
+    If using a different version, you can try modifying the version parameters
+    in the URL.
+    
+    c. If regenerating the Gate Client API produced any changes, kick off the
+    Flow_BuildAndValidate_1.xx.x for the release branch and wait for a successful
+    completion. This will trigger a downstream Build_PrimaryArtifacts job that
+    we rely on later.
+    
+    d. Run Publish_SpinRelease with the following parameters:
+    
+    - SPIN_BUILD_VERSION_TO_RELEASE: This can be found in the build_spin files
+    written by the Build_PrimaryArtifacts job. Use the version found in the most
+    recent run of the Build_PrimaryArtifacts for the release branch. Note: The
+    major minor part of this version number should match the Gate version for
+    the release branch. If it does not, double check that a tag for the previous
+    minor version of Spin CLI exists. The build auto increments new tags based
+    on the highest pre-existing minor tag.
+    
+    - BOM_VERSION: This is the BOM to associate the Spin CLI release with. It is
+    the latest Spinnaker release number, 1.xx.x.
+
+10. Publish a Sponnet minor version. Run the [publish.sh](https://github.com/spinnaker/spinnaker/blob/master/sponnet/publish.sh)
```

Comment:
@ttomsu This link is no longer valid, I'm guessing because it got moved into the sponnet project, but just wanted to double-check with you that this is the same `publish.sh`: https://github.com/spinnaker/sponnet/blob/master/publish.sh

Reply:
Sure looks right to me.